src/verify.c

```diff
@@ -474,6 +478,30 @@ static void add_missing_files(struct manifest *official_manifest)
 		if ((ret != 0) || hash_needs_work(file, local.hash)) {
```

Comment:
I tried to trigger this error on a tmpfs filesystem and I couldn't do it. All files were created empty in the filesystem after the disk got full, so hash_needs_work returns false, even for files that weren't installed successfully.
Could you get into this if using swupd verify --install?

Reply:
Nice. I'll test that. Maybe the behavior when ramfs is used is different.

Reply:
It's working fine with file mounted partitions, so I think we're good about that.